app/src/main/java/com/example/dinhh/soundscape/data/Model.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.dinhh.soundscape.data
+
+import com.example.dinhh.soundscape.presentation.ListItem
+
+object Model {
+    val category: kotlin.collections.MutableList<ListItem> = java.util.ArrayList()
+    val sounds: kotlin.collections.MutableList<ListItem> = java.util.ArrayList()
+
+
+    init {
+        category.add(ListItem("Nature"))
```

Comment:
It should be enum

Reply:
What should be enum? Can you be more precise?

Reply:
The category should be enum. For example 

enum Category {
 NATURE,
 HUMAN,
 ....
}

But we can leave it for now. We can refactor the code later on

Reply:
Lets check this out together tomorrow. I'm not familiar with enums at all.